packages/amplify_authenticator/lib/src/widgets/form_field.dart

```diff
@@ -175,13 +174,16 @@ abstract class _AuthenticatorFormFieldState<FieldType,
   /// Maximum number of lines to use for error text.
   int get errorMaxLines => 1;
 
+  /// Value used by stateful widgets which require state for selected values
+  FieldValue? selectionValue;
```

Comment:
Only needed in specific form fields - I would just keep it there. Or better yet, in a radio group like before. Most form fields do not control their own selection - they belong to a set of form fields which together are told their selection by a parent.